notebooks/243-tflite-selfie-segmentation/243-tflite-selfie-segmentation.ipynb

```diff
@@ -0,0 +1,707 @@
+{
```

Comment:
<p>Should we clear cell outputs before merge?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
usually, we prefer to stay output to make it more friendly for users to see expected result without running notebook